ang/civicase/CaseDetailsPeopleTab.js

```diff
@@ -79,11 +79,24 @@
       if (onlyChecked || $scope[tab + 'SelectionMode'] === 'checked') {
         return _.collect(_.filter($scope[tab], {checked: true}), idField);
       } else if ($scope[tab + 'SelectionMode'] === 'all') {
-        return _.collect($scope[tab], idField);
+        return _.collect(_.filter($scope[tab], function (el) {
+          return el.contact_id;
```

Comment:
does it mean that a contact might not have a `contact_id` property?

Reply:
Sorry, but $scope[tab] contains the list of the roles. which need to be filtered again to only those roles which are assigned to any contact.